src/trusted/service_runtime/sel_ldr.c

```diff
@@ -57,8 +59,28 @@
 #include "native_client/src/trusted/simple_service/nacl_simple_service.h"
 #include "native_client/src/trusted/threading/nacl_thread_interface.h"
 
+double time_counter = 0.0;
+double time_start = 0.0;
+double time_end = 0.0;
+int cage = 1001;
```

Comment:
I believe this (cage = 1001) isn't used any more. Please help remove it.

Reply:
Indeed it's not, will remove.